File: cpp/09proc/Proc21.cpp
#include <iostream>
using namespace std;

int SumRange(int A, int B);

int main()
{
	//Task("Proc21");
	int A, B, C;
	cin >> A >> B >> C;

	cout << SumRange(A, B);
	cout << SumRange(B, C);
	
	return 0;
}

int SumRange(int A, int B)
{
	int sum = 0;
	for (int i = A; i <= B; i++)
		sum += i;
	
	return sum;
}

File: php/09proc/proc21.php
<?php
function SumRange($A, $B){
        $sum = 0;
        if($A < $B)
			for($i = $A; $i <= $B; $i++)
                $sum += $i;
        return $sum;
}
$A = mt_rand(1, 100);
$B = mt_rand(1, 100);
$C = mt_rand(1, 100);
echo "A = ".$A;
echo "<br>B = ".$B;
echo "<br>C = ".$C;
$AB = SumRange($A, $B);
$BC = SumRange($B, $C);
echo "<br>Сумма аз A то B: ".$AB;
echo "<br>Сумма аз B то C: ".$BC;
?>